web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.html

```diff
@@ -77,16 +77,17 @@
 	</div>
 
 	<div class="flex items-center relative">
-		<label class="text-16 text-new.gray-500 block w-full capitalize whitespace-nowrap mr-6px" for="tailMode">Tail Events</label>
+		<label class="text-12 text-new.gray-500 block w-full capitalize whitespace-nowrap mr-6px" for="tailMode">Tail Events</label>
 		<input
+			#tailModeToggle
 			type="checkbox"
 			class="peer w-24px h-18px border border-new.primary-50 rounded-4px checked:border-new.primary-400 checked:bg-new.primary-25 focus:bottom-2 focus:border-new.primary-400 checked:bg bg-white-100"
 			name="enableTailMode"
 			[checked]="enableTailMode"
 			id="tailMode"
 			(change)="toggleTailMode($event)"
 		/>
-		<img src="assets/img/check-icon-primary.svg" class="w-12px hidden peer-checked:block absolute top-[26%] right-[3px]" alt="check icon" />
+		<img src="assets/img/check-icon-primary.svg" class="w-12px hidden peer-checked:block absolute top-[20%] right-[3px]" alt="check icon" (click)="tailModeToggle.click()" />
```

Comment:
This shouldn't require a click here. Doesn't the checkbox support uncheck? @Oluwadaminiola